knative-operator/pkg/common/serving/openshift.go

```diff
@@ -1,4 +1,4 @@
-package common
+package serving
```

Comment:
I'm struggling with this, honestly. I kinda don't want to reorganize the common dir until we think we have something eventing-related that should go in here.

Reply:
@jcrossley3 fair enough. let's keep this PR as is until the eventing related code is free to go into this repo.